src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -522,8 +522,9 @@ private static GeneratableElement getExampleFor(ModElement modElement, boolean u
 			components.add(new Label("text2", 100, 150, new StringProcedure(!_true ? "string2" : null, "fixed value 2"),
 					Color.white, new Procedure("condition4")));
 
-			components.add(new Image(20, 30, "pricture1", true, new Procedure("condition1")));
-			components.add(new Image(22, 31, "pricture2", false, new Procedure("condition2")));
+			components.add(new Image(20, 30, "pricture1", true, new Procedure("condition1"), new Procedure("number1")));

```

Comment:
There should be case where number is not provided, eg. image is not animated as this is a valid test case

Reply:
So a `null` procedure?

Reply:
✔️ Fixed